app/src/components/Dropdown.tsx

```diff
@@ -3,29 +3,37 @@ import { ChangeEvent, ReactElement } from 'react'
 
 import Required from '@components/Required'
 
-interface Props<T> {
-  handleChange: (
-    e: ChangeEvent<HTMLSelectElement> & { target: { value: T } }
-  ) => void
+import { i18nKey } from '@src/types'
+
+export interface DropdownProps {
+  handleChange: (e: ChangeEvent<HTMLSelectElement>) => void
```

Comment:
Notes for reviewers: The main reason this component is included as part of this PR and not an earlier PR in the series is because of this type change.